src/landOnFunctions.js

```diff
@@ -1,30 +1,119 @@
 export function standardProperty(activePlayer, emitter){
     if(this.ownerID === null){
-        emitter.emit(`promptBuy${activePlayer.socketID}`, this, activePlayer);
+        emitter.once(`confirmBuy`, (io, bool) => {
+            if(bool){
```

Comment:
Keywords such as if/else/return, we typically make sure there's a space after them.  That way it's clear that we're using a keyword and not calling a function.